mbuild/conversion.py

```diff
@@ -716,6 +774,39 @@ def from_pybel(pybel_mol,
     return compound
 
 
+def from_rdkit(rdkit_mol,
+               compound=None,
+               coords_only=False,
+               smiles_seed=0):
+    from rdkit import Chem
+    from rdkit.Chem import AllChem
+
+    mymol = Chem.AddHs(rdkit_mol)
+    AllChem.EmbedMolecule(mymol, randomSeed=smiles_seed)
+    single_mol = mymol.GetConformer(0)
+    xyz = single_mol.GetPositions()
```

Comment:
Should divide `xyz` by 10 to get the positions in nanometers

Reply:
Good catch! Will fix that soon!